pkg/operators/kubemanager/kubemanager.go

```diff
@@ -310,6 +312,9 @@ func (m *KubeManagerInstance) PostGadgetRun() error {
 }
 
 func (m *KubeManagerInstance) enrich(ev any) {
+	if event, canEnrichEventWithNode := ev.(operators.ContainerNodeSetter); canEnrichEventWithNode {
+		m.manager.gadgetTracerManager.ContainerCollection.EnrichEventWithNode(event)
+	}
```

Comment:
@flyth pointed me to this discussion for the node enrichment:
https://github.com/inspektor-gadget/inspektor-gadget/pull/1370#discussion_r1149464492

We should decide on one of the implementation but not both.

Reply:
If there is no opinions on this and since #1370 (bpftrace) is not ready to be merged, and this PR is almost ready, I suggest to keep this change as is. It can always be changed to the other implementation if the node enricher in #1370 (bpftrace) is preferred. What do you think?

Reply:
Sure! That discussion should not block the development here.